RecoMuon/MuonIsolation/plugins/ExtractorFromDeposits.h

```diff
@@ -26,6 +26,7 @@ class ExtractorFromDeposits : public reco::isodeposit::IsoDepositExtractor {
       const reco::TrackCollection & tracks) override;
   reco::IsoDeposit deposit (const edm::Event & ev, const edm::EventSetup & evSetup,
       const reco::Track & track) const override;
+
   virtual reco::IsoDeposit deposit (const edm::Event & ev, const edm::EventSetup & evSetup,
       const reco::TrackRef & track) const;
```

Comment:
actually, it would be nicer to get rid of any changes in this file
so that this PR does not interfere with my update